generator/src/googleapis/codegen/languages/java/2.0.0/features.json

```diff
@@ -0,0 +1,487 @@
+{
+  "language": "java",
+  "description": "Java libraries for Google APIs.",
+  "generator": "java1_15",
+  "releaseVersion": "1.32.1",
+  "baseClientLibrary": "2.0.0",
```

Comment:
Set to be v2.0.0 from api-client

Reply:
The other Google libraries have also been updated

Reply:
@suztomo Should we try to match these versions to whatever is in the latest shared-dependencies release? This would help avoid diamond dependency conflicts between Apiary and GAPIC.

Reply:
That's a nice practice we can follow. How about "_the shared dependencies BOM version that was used for the latest Google Cloud BOM release_". This ensures the unit and integration tests of the client libraries (though these are based on gRPC, not Apiary) pass with the set of the library versions in the shared dependencies BOM.

Reply:
> _the shared dependencies BOM version that was used for the latest Google Cloud BOM release_

It was wrong. This would create a circular dependency:

- google-cloud-bom includes google-cloud-bigquery
- google-cloud-bigquery uses google-api-services-bigquery.
- google-api-services-bigquery uses google-api-client.
- My suggestion above was requiring google-api-services-bigquery's google-api-client version to be the one used in google-cloud-bom release.

Reply:
So for now, let's use the versions in the shared dependencies BOM as much as possible:

- https://github.com/googleapis/java-shared-dependencies/blob/v2.13.0/third-party-dependencies/pom.xml
- https://github.com/googleapis/java-shared-dependencies/blob/v2.13.0/first-party-dependencies/pom.xml

Reply:
Sure, updated the versions that I could match in the poms. Unable to find jetty, jdo, and transationsApi